docs/intro/cli.rst

```diff
@@ -0,0 +1,128 @@
+.. _ref_intro_cli:
+
+.. _ref_admin_install:
+
+=======
+The CLI
+=======
+
+The ``edgedb`` command line tool is an integral part of the developer workflow
+of building with EdgeDB. Below are instructions for installing it.
+
+Installation
+------------
+
+To get started with EdgeDB, the first step is install the ``edgedb`` CLI.
+
+**Linux or macOS**
+
+.. code-block:: bash
+
+    $ curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh
+
+**Windows Powershell**
+
+.. code-block:: powershell
+
+    PS> iwr https://ps1.edgedb.com -useb | iex
+
+Follow the prompts on screen to complete the installation. The script will
+download the ``edgedb`` command built for your OS and add a path to it to your
+shell environment. To test the installation, run ``edgedb --version`` from the
+command line.
+
+.. code-block:: bash
+
+    $ edgedb --version
+    EdgeDB CLI 1.x+abcdefg
+
+If you encounter a ``command not found`` error, you may need to open a new
+terminal window before the ``edgedb`` command is available.
+
```

Comment:
```suggestion

If issues are still present, you may need to check your PATH environment variables to ensure
``edgedb`` is properly pointing to the source directory.

```

This is Windows OS specific, I've ran into this before and people may not realise that the PATH env var is not properly fitted for the terminal. Just a proposal, might not be needed at all to explain.